### PR TITLE
Minor readme update

### PR DESCRIPTION
**Pull request by cancelei**
Hello reviewer, 

**-- What animals belong to Melody Pond?**
image

**-- List of all animals that are Pokemon (their type is Pokemon).**
image

**-- List all owners and their animals, remember to include those that don't own any animal.**
image

**-- How many animals are there per species?**
image

**-- List all Digimon owned by Jennifer Orwell.**
image

**-- List all animals owned by Dean Winchester that haven't tried to escape.**

All animals owned by Dean have tried to escape. **So the table would be empty.** The print screen below is just to show how many animals Dean owns.

image

**-- Who owns the most animals?**
image

Happy review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,12 +113,10 @@ To run the project:
```
Comment (Munsa1) [inline README.md:87]: - [x] Consider removing the link to the `FAQ` section as it does not exist making the link broken.
image

File: README.md [hunk 2]
```diff
@@ -108,12 +113,10 @@ To run the project:
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (Munsa1) [inline README.md:89]: - [ ] Kindly remove comments from your readme file for a more professional readme
image

File: README.md [hunk 3]
```diff
@@ -68,6 +68,11 @@ After you're finished please remove all the comments and instructions!
 - **Updated Vet Data Base**
 - **Queries to filter from data set**
 - **Schema for the data set**
+- **Use primary key & foreign key mechanism for joining tables.**
```
Comment (Munsa1) [inline README.md:45]: - [ ] Please fix the link to your `LinkedIn` profile as it  is currently showing a 404 page
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @cancelei, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added Screenshots :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 
Correct Results From Queries ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @cancelei, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed unused FAQ link in menu. ✔️ 
- Removed unused comments in the readme file. ✔️ 
- Fixed the link to LinkedIn profile. ✔️ 
 
## [Optional] suggestions:
👉 One optional suggestion for improvement is to follow the convention for naming your branch. Usually, branch names should use the past tense verb to indicate that the feature has been implemented. For example, instead of `query_multiple_tables`, you could use `queried_multiple_tables`. This would make your branch name more consistent and clear for future pull requests.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**